CMakeLists.txt

```diff
@@ -670,6 +670,20 @@ endif(${WITH_LTTNG})
 
 option(WITH_OSD_INSTRUMENT_FUNCTIONS OFF)
 
+#option for eBPF tracing (cephtrace)
+option(WITH_CEPHTRACE "Build eBPF tracing tools (radostrace)" ON)
```

Comment:
could make `WITH_CEPHTRACE` an dependent option, see https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html. like
```cmake
CMAKE_DEPENDENT_OPTION(WITH_CEPHTRACE "Build eBPF tracing tools (radostrace)" ON
  LINUX OFF)
```
so that, when `LINUX` is true, this option is `ON` by default, otherwise it's set to `OFF` and hidden from the user.

Reply:
Thanks! let me do that